test/Driver/loaded_module_trace.swift

```diff
@@ -34,3 +35,5 @@
 // CHECK-CONFIRM-ONELINE: {"name":{{.*}}]}
 
 import Module2
+
+@freestanding(expression) macro echo<T>(_: T) -> T = #externalMacro(module: "Plugin", type: "EchoMacro")
```

Comment:
This is required to actually load the plugin, otherwise plugin dependencies are not recorded.

Reply:
Would it be better if you put this declaration in Inputs/loaded_module_trace_compiler_plugin.swift, so that this is imported from a module like an actual macro adopter library would end up?